src/main/java/net/tridentsdk/meta/item/ItemDisplayProperties.java

```diff
@@ -54,6 +55,39 @@
      * @param lore Lore you wish to set it to
      */
     default void setLore(String... lore) {
-        setLore(Arrays.asList(lore));
+        this.setLore(Arrays.asList(lore));
+    }
+
+    /**
+     * Adds lore to the item starting from a specific index.
+     *
+     * @param index Index to begin insertion at. If equal to number of lore, it appends.
+     * @param lore Lore to be added.
+     *
+     * @see #addLore(String...)
+     */
+    default void addLore(int index, String... lore) {
+        List<String> curr = this.lore();
+        if (curr == null) {
+            curr = Lists.newArrayList();
+        }
+        if (curr.isEmpty() || index == curr.size()) {
```

Comment:
Yes, addLore is intended to add, not replace. setLore would be used in that use-case.